libs/language-server/src/stdlib/domain/geo/ContryCode.jv

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: 2024 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use * from "./CountryCodeAlpha2.jv";
+use * from "./CountryCodeAlpha3.jv";
+use * from "./CountryCodeNumeric.jv";
+
+/**
+* Any code (alppha2, alpha3, numeric) for countries as defined by ISO 3166.
+*/
+publish valuetype CountryCode oftype text {
+  constraints: [
+    CountryCodeAlpha2Constraint,
+    CountryCodeAlpha3Constraint,
+    CountryCodeNumericConstraint
```

Comment:
Ah that might be a mistake because it might join them with an "AND"

Reply:
Indeed. I think this needs to be removed.